source/ssl/include/ssl.ooc

```diff
@@ -0,0 +1,45 @@
+/* This file is part of magic-sdk, an sdk for the open source programming language magic.
```

Comment:
So `source/ssl` contains... just a folder? I first of all think the folder should rather be named `external` (like in `system`), but more importantly I'd be OK with just having this file reside in `source/ssl` directly.